draft-ietf-quic-multipath.md

```diff
@@ -1382,6 +1382,31 @@ Maximum Path Identifier:
   Receipt of a value that is higher than the local maximum value MUST
   be treated as a connection error of type PROTOCOL_VIOLATION.
 
+
+## PATH_CID_BLOCKED frames {#path-cid-blocked-frame}
```

Comment:
Is this PATH_CID_BLOCKED or PATH_CIDS_BLOCKED?

We use plural for PATHS_BLOCKED to align with STREAMS_BLOCKED in rfc9000

Reply:
Updated.